cli/run/import-ts-plugin.ts

```diff
@@ -0,0 +1,11 @@
+import { resolve } from 'path';
+
+export default async (configPath: string) => {
```

Comment:
just using import() doesn't work, so i added this file which will be transformed into a normal node require import by the new build plugin i wrote.

Reply:
I would be interested to know why it would not work. Did you mark the plugin as external in the Rollup config? Rollup should automatically translate a dynamic import into a require wrapped in a promise if done right. The default export of the library should then be the default property of the resolved object

Reply:
i did add the plugin to [the external array]( https://github.com/rollup/rollup/blob/51e727c99bfc67a6bc46087c63950cec2a7fe12f/rollup.config.js#L94), the reason used babel is because in the compiled code it still used a dynamic import, i will try again maybe it will work this time.

Reply:
See my other comment, it is because you did not pass a static string to the import.